dashboard/client/src/pages/job/JobTaskNameProgressTable/JobTaskNameProgressTable.tsx

```diff
@@ -68,32 +69,21 @@ export const JobTaskNameProgressTable = ({
           </TableRow>
         </TableHead>
         <TableBody>
-          {progress.map(({ name, progress }) => {
-            const {
-              numFailed = 0,
-              numPendingArgsAvail = 0,
-              numPendingNodeAssignment = 0,
-              numRunning = 0,
-              numSubmittedToWorker = 0,
-            } = progress;
-
-            const numActive =
-              numPendingArgsAvail +
-              numPendingNodeAssignment +
-              numRunning +
-              numSubmittedToWorker;
-
-            return (
-              <TableRow>
-                <TableCell align="center">{name}</TableCell>
-                <TableCell align="center">{numFailed}</TableCell>
-                <TableCell align="center">{numActive}</TableCell>
-                <TableCell>
-                  <MiniTaskProgressBar {...progress} />
-                </TableCell>
-              </TableRow>
-            );
-          })}
+          {progress.map(
+            ({ name, progress, numFailed, numActive, numFinished }) => {
+              return (
+                <TableRow>
+                  <TableCell align="center">{name}</TableCell>
+                  <TableCell align="center">{numFailed}</TableCell>
+                  <TableCell align="center">{numActive}</TableCell>
```

Comment:
nit: my personal preference is actually showing active desc first, then failed. if this is part of the PRD i'm ok leaving as is

Reply:
I'm showing num failed first because that is the sort order. It prioritizes the tasks with the most failures first!